client/components/jetpack/daily-backup-status/status-card/backup-scheduled.jsx

```diff
@@ -86,7 +86,7 @@ const BackupScheduled = ( { lastBackupDate } ) => {
 						),
 					},
 				} ) }
-			</div>
```

Comment:
This didn't have `preventWidows` but for consistency with the above components I changed the markup here too. It appears before a backup is started:

<img width="293" alt="Screenshot 2025-05-08 at 3 03 13 pm" src="https://github.com/user-attachments/assets/2574fd9f-f0ab-4444-a9e6-9638b867290c" />